plugin-gradle/README.md

```diff
@@ -129,6 +134,45 @@ spotless {
 }
 ```
 
+<a name="groovy"></a>
+
+## Applying to Groovy source
+
+Configuration for Groovy is similar to [Java](#java).  Most java steps, like `licenseHeader` and `importOrder`, support Groovy as well as Java.
+
+The groovy formatter's default behavior is to format all `.groovy` and `.java` files found in the Groovy source directories.  If you would like to exclude the `.java` files, set the parameter `excludeJava` to `true`, or you can set the `target` parameter as described in the [Custom rules](#custom) section.
+
+```gradle
+apply plugin: 'groovy'
+...
+
+spotless {
+	java {
+		licenseHeaderFile 'spotless.license.java'
+		googleJavaFormat()	//Use specific formatter for Java files
+	}
+	groovy {
+		licenseHeaderFile 'spotless.license.java' // License header file
+		excludeJava // Exclude the java
+		//The Groovy-Eclipse formatter step basically extends the JDT formatter.
+		//It can format Java files as well as Groovy files
+		greclipseFormatFile 'greclipse.properties'
+	}
+}
+```
```

Comment:
I want to touch them up such that the code block looks like this:

```gradle
spotless {
 	java {
 		licenseHeaderFile 'spotless.license.java'
 		googleJavaFormat() // use a specific formatter for java files
 	}
 	groovy {
 		licenseHeaderFile 'spotless.license.java'
 		excludeJava // Excludes all java sources within the Groovy source dirs from formatting
 		
		// the Groovy-Eclipse formatter extends the Java Eclipse formatter,
 		// so it formats Java files by default (unless `excludeJava` is used).
 		greclipseFormatFile 'greclipse.properties'
 	}
 }
```

What do you think?

If you think this change is good, would you kindly push it for me? :)